docs/Contributing/ContributingCode.mdr

```diff
@@ -1,3 +1,56 @@
 # Contributing Code
 
-> stub, to be filled
\ No newline at end of file
+> this is a stub for now, this place deserves better
+
+You might be here because you found something you could improve in [Gwion](https://github.com/fennecdjay/gwion).
+Please open an [issue](https://github.com/fennecdjay/gwion/issues) describing the problem and how you might address it.
+
+## Dependencies
+
+-
+  
+## Editing the code
+
+### Fork the repository
+
+You can do this through the github site, or command-line tools like `gh` or `hub`. See **Further Reading**.
+
+### Clone the source
+
+Clone the url of your fork
+
+``` sh
+git clone https://github.com/<your username>/gwion
+```
+
+### Edit some files
+
+``` sh
+cd gwion
+make watch
+```
+
+### Add your changes
+
+``` sh
+git add docs/myfile.mdr
```

Comment:
Shouldn't this be more like
``` sh
git add src/myfile.c
```
?

Reply:
Also correct. I'll fix that real quick.

Reply:
Done.

Reply:
Nice.
Is it OK to merge for you, or is there anything left?

Reply:
No, I think that's it. Thank you!

Reply:
Merging now.
Nice Work :smiley: